src/plugins/ui_core_views/evaluation_chart.ts

```diff
@@ -104,10 +105,22 @@ export class EvaluationChartPlugin extends UIPlugin<EvaluationChartState> {
         const figureId = figure.id;
         const figureData = this.getters.getChart(figureId)?.getDefinitionForExcel();
         if (figureData) {
-          figures.push({
-            ...figure,
-            data: figureData,
-          });
+          // Converts the chart title as cell value if it is a reference
+          if (figureData.titleType === "reference") {
+            const range = this.getters.getRangeFromSheetXC(sheet.id, figureData.title!);
+            if (range === undefined || range.invalidXc || range.invalidSheetName) {
+              throw new InvalidReferenceError();
+            }
```

Comment:
I think we'd want to set the title to an empty string rather than throwing and preventing the export